src/App/Containers/BooksForm.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-unused-vars */
+import { useState } from 'react';
+import Select from 'react-select';
+
+const bookCategories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+const options = bookCategories.map((category) => ({
+  value: category,
+  label: category,
+}));
+
+const handleSubmit = (e) => {
+  e.preventDefault();
+  console.log('e');
```

Comment:
- [ ] [REQUIRED] No console.log committed to repo. It's ugly, it kills performance. and it can make confidential data be visible in the browser tools. (screenshot from Pathwright)